ms/MSOper/MSMetaData.cc

```diff
@@ -3399,6 +3399,8 @@ void MSMetaData::_computeScanAndSubScanProperties(
     uInt chunkSize = nrows/nchunks;
     if (nrows % nchunks > 0) {
         ++chunkSize;
+        // integer division
+        nchunks = nrows/chunkSize + 1;
     }
```

Comment:
Actually, the unit test fails when I try that substitution, so it appears I wasn't correct :) .